src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -319,7 +319,8 @@ function enableQuestion(questionNum) {
     $('#' + FEEDBACK_QUESTION_EDITTYPE + '-' + questionNum).val('edit');
     $('#button_question_submit-' + questionNum).show();
 
-    showVisibilityCheckboxesIfCustomOptionSelected($currentQuestionTable);
+    $currentQuestionForm = $currentQuestionTable.closest('form');
```

Comment:
Is this variable defined before? May need to add `var $currentQuestionForm = ...` if not.